assets/js/components/module/ModuleApp.js

```diff
@@ -43,14 +43,15 @@ import HelpMenu from '../help/HelpMenu';
 import DateRangeSelector from '../DateRangeSelector';
 import HelpMenuLink from '../help/HelpMenuLink';
 import { CORE_MODULES } from '../../googlesitekit/modules/datastore/constants';
+import ProgressBar from '../ProgressBar';
 
 const { useSelect } = Data;
 
 function ModuleApp( { moduleSlug } ) {
 	const helpVisibilityEnabled = useFeature( 'helpVisibility' );
 	const screenWidgetContext = useSelect( ( select ) => select( CORE_MODULES ).getScreenWidgetContext( moduleSlug ) );
 	const moduleConnected = useSelect( ( select ) => select( CORE_MODULES ).isModuleConnected( moduleSlug ) );
-	const shouldRenderWidget = useFeature( 'widgets.moduleScreens' ) && screenWidgetContext;
+	const shouldRenderWidget = useFeature( 'widgets.moduleScreens' );
```

Comment:
This is a bit of a convention with `useFeature` – if you look at other uses of the hook, you'll see the variable is `const {feature}Enabled = useFeature(...)` so it would be great to preserve that for consistency.

Reply:
@aaemnnosttv That sounds really obvious now you've pointed it out! :+1:  

Changed.  I'm using feature flags on other tasks as well so I'll make sure I make the same changes